test/python/test_secrets_api.py

```diff
@@ -266,17 +266,21 @@ def test_get_variables_422(self):
 
         self.assertEqual(context.exception.status, 422)
 
-    def test_get_variables_encoded(self):
+    def test_get_variables_encoded(self): # pylint: disable=R1710
```

Comment:
can you add some context about what you are disabling here?

Reply:
Removed based on test rework